client/src/components/Bookings/BookingCard/useStyles.ts

```diff
@@ -0,0 +1,25 @@
+import { makeStyles } from '@material-ui/core/styles';
+
+const useStyles = makeStyles(() => ({
+  root: {
+    padding: 18,
+  },
+  date: {
+    fontSize: 16,
+    fontWeight: 500,
+  },
+  user: {
+    fontSize: 16,
+    fontWeight: 700,
+  },
+  status: {
+    fontWeight: 700,
```

Comment:
Consider using Typography variants, that might do the trick as opposed to using CSS classes to set font weight and sizes :)

Reply:
I moved all typography to the default theme with the variants